### PR TITLE
MIST-389 Updating READMEs

### PR DESCRIPTION
**Pull request by TauZero**
[Review on Reviewable](https://reviewable.io/reviews/mistifyio/mistify-agent/38)

### CODE REVIEW

**Comment by mmlb** [comment]
<sup>**[cmd/mistify-agent/doc.go, line 2 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnNnF7ZDPH7tWGTfQa)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/cmd/mistify-agent/doc.go#L2)):</sup>
'.' @ EOL

---

<sup>**[doc.go, line 7 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnO2heHw1y65P6CR6f)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/doc.go#L7)):</sup>
Maybe this "core" part should be in the synopsis?

---

<sup>**[doc.go, line 8 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnOGL2i-f9Ta7ucx_O)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/doc.go#L8)):</sup>
Maybe reword:

a hypervisor and defines a general set of actions. Mistify Agent works mostly as an interface to the sub-agents, which do most of the work.

---

<sup>**[doc.go, line 32 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnRXgfX-Q2okke0KTb)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/doc.go#L32)):</sup>
performed configurable
                     ^ is

---


---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38)

**Comment by TauZero** [comment]
<sup>**[cmd/mistify-agent/doc.go, line 2 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnNnF7ZDPH7tWGTfQa)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/cmd/mistify-agent/doc.go#L2)):</sup>
Done.

---

<sup>**[doc.go, line 7 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnO2heHw1y65P6CR6f)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/doc.go#L7)):</sup>
Done.

---

<sup>**[doc.go, line 8 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnOGL2i-f9Ta7ucx_O)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/doc.go#L8)):</sup>
Done.

---

<sup>**[doc.go, line 32 \[r1\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmnRXgfX-Q2okke0KTb)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/adba4d03c270e92a03dc099c088fbf0185bc514a/doc.go#L32)):</sup>
Done.

---


---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38)

**Comment by mmlb** [comment]
Reviewed files:
- .godocdown.template @ r1
- client/client.go @ r1
- cmd/mistify-agent/doc.go @ r2
- config/config.go @ r1
- config/README.md @ r1
- doc.go @ r2
- examples/simple-subagent/doc.go @ r3
- examples/simple-subagent/main.go @ r3
- examples/sv/agent.json @ r3
- examples/sv/log @ r3
- examples/sv/README.md @ r3
- examples/sv/run @ r3
- examples/test-rpc-service/doc.go @ r3
- examples/test-rpc-service/README.md @ r3
- examples/webhook-post/doc.go @ r3
- examples/webhook-post/main.go @ r3
- examples/webhook-post/README.md @ r3
- rpc/README.md @ r1
- rpc/rpc.go @ r1

---

<sup>**[client/README.md, line 1 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtABoIH1dNcFPcfM0D)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/client/README.md#L1)):</sup>
Did you forget to add/push client/doc.go?

---

<sup>**[cmd/mistify-agent/README.md, line 5 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtAbxKhnd8NyZt7B7k)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/cmd/mistify-agent/README.md#L5)):</sup>
Seems out of date, this should have had a `.` at the end no?

---

<sup>**[examples/simple-subagent/README.md, line 1 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtBf2XrQgKY0B1357u)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/examples/simple-subagent/README.md#L1)):</sup>
I think this is out of date

---

<sup>**[README.md, line 1 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtB3nFwc6V4qssO_IC)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/README.md#L1)):</sup>
I think this file is out of date.

---


---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38)

**Comment by TauZero** [comment]
<sup>**[client/README.md, line 1 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtABoIH1dNcFPcfM0D)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/client/README.md#L1)):</sup>
I didn't think it needed much more than the package line and godocs, so didn't create a separate `doc.go`. Just generated from the existing package comment.

---

<sup>**[cmd/mistify-agent/README.md, line 5 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtAbxKhnd8NyZt7B7k)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/cmd/mistify-agent/README.md#L5)):</sup>
Forgot to regenerate after adding period. Done.

---

<sup>**[examples/simple-subagent/README.md, line 1 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtBf2XrQgKY0B1357u)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/examples/simple-subagent/README.md#L1)):</sup>
This one actually isn't generated. Actual markdown hierarchical headers, go syntax highlighting, etc. seemed to improve readability greatly. The `doc.go` is for the actual package documentation and that has been copied over to this README.

---

<sup>**[README.md, line 1 \[r3\]](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38#-JmtB3nFwc6V4qssO_IC)** ([raw file](https://github.com/mistifyio/mistify-agent/blob/2017f2668f32e657cc743847ecf8f1553ccdf16d/README.md#L1)):</sup>
Forgot to regenerate. Done.

---


---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38)

**Comment by mmlb** [comment]
Reviewed files:
- client/README.md @ r1
- cmd/mistify-agent/README.md @ r4
- examples/simple-subagent/README.md @ r3
- README.md @ r4

---





Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/mistifyio/mistify-agent/38)
